modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,36 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+== packagemanifests
+
+`run packagemanifests` subcommand deploys an Operator's package manifests with OLM. The command argument must be set to a valid package manifest root directory, for example, ``<project-root>/packagemanifests`.
```

Comment:
Please check if the two backticks at the beginning of "\<project-root\>" are intentional.

Also, the doc guidelines for [code blocks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) indicate to use underscores between words rather than hyphens. (It's scroll-o-rama from the link, Ctrl+F for underscore after you click it.)